website/docs/r/app_service.html.markdown

```diff
@@ -154,6 +154,7 @@ The following arguments are supported:
 `site_config` supports the following:
 
 * `always_on` - (Optional) Should the app be loaded at all times? Defaults to `false`.
+* `app_command_line` - (Optional) App command line to launch, e.g. `/sbin/myserver -b 0.0.0.0`.
```

Comment:
since the schema's shared between the App Service and App Service Slot resources - could we update the docs for App Service Slot too?